iree/tools/iree-benchmark-module-main.cc

```diff
@@ -230,7 +142,70 @@ int main(int argc, char** argv) {
       absl::flags_internal::OnUndefinedFlag::kIgnoreUndefined);
   ::benchmark::Initialize(&argc, argv);
   iree::InitializeEnvironment(&argc, &argv);
-  IREE_CHECK_OK(iree::RegisterModuleBenchmarks());
+
+  IREE_CHECK_OK(iree_hal_module_register_types());
```

Comment:
Could you move this back into a separate function? This is pretty noise for main IMO and also defeats status usage

Reply:
By this do you mean everything (creating device, module...etc) ?  `IREE_CHECK_OK(...);` asserts prints which like so its actually clear what is going on when it fails.

Reply:
Yes. Having it in a separate function was deliberate to allow for status usage (and avoid a giant main function). Now you've got manually returning exit codes